main.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 // Version holds the current Gitea version
-const Version = "0.9.99.0915"
```

Comment:
I think `Var Version = "0.9.0+dev"` is better. So that if user `go get` then the version is shown it's a dev version.